index.html

```diff
@@ -13,139 +13,154 @@
 </head>
 
 <body>
-    <header class="container-header">
-        <a id="name" href=""><img class="mylogo" src="attributes/mylogo.png" alt="logo" /></a>
-        <nav><img class="union" src="attributes/union.png" alt="union_icon" /></nav>
-        <nav class="desktop_headicons">
-            <a class="header_btn" href="">Portfolio</a>
-            <a class="header_btn" href="#about_text">About</a>
-            <a class="header_btn" href="#full_name">Contact</a>
-            <a class="header_btn" href=""><img id="mail" src="attributes/mailicon.png" alt="mailicon" /></a>
+    <header>
+        <nav class="desktop-nav">
+            <a id="name" href=""><img class="mylogo" src="attributes/mylogo.png" alt="logo" /></a>
+            <ul class="main-menu">
+                <li class="nav-item"><a class="header-btn" href="">Portfolio</a></li>
```

Comment:
- [ ] This link is supposed to take us to the works section. Kindly update the `href` accordingly 👍